client/components/Account/AccessList.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <fieldset>
+    <legend>People with access to the shared account</legend>
+    <table>
+      <tr v-for="username in $store.getters.credentials"
+          :key="username">
+        <td>
+          @{{username}}
+        </td>
+        <td>
+          <button class="btn-danger"
+                  @click="() => deleteCredential(username)">
+            Remove from shared account
```

Comment:
I also believe if you revoke access to a shared account, you are also essentially deleting that user account. Perhaps rather than just "Remove from shared account" there should be some language also stating that user's account will also be deleted? 

In general, I realize our design does not allow the same user to be part of two shared accounts, so a user with two elderly parents, for example, would have to have two separate accounts for both of their parent's shared accounts. This might be less of an issue for the language of "removing and deleting" being clear since a user's account wouldn't accidentally be deleted from another shared account, but it does speak to a greater design flaw of our application. 

Not sure if there's any actionable item to be done here, but maybe something to mention in the design revision?

Reply:
removing access only affects users and never delete any accounts. I'm not exactly sure what you are saying here, but yes this particular action is probably a bit more prickly than it should be.

Reply:
down for design revision and stretch goal for not allowing revoking access when there's one user left

I do think in a sense, you can have the same usual "account" concept for both parents if you add the same credentials for both, but it's weird because these are actually just two different sets of credentials whose updates are not boound to each other, though it may give the illusion that they're the same. Don't think there's a quick fix other than adding more instructions/general explanation on the account pages?

Reply:
yeah it's definitely a deeper design flaw. Including in the design revision makes sense to me. I won't have time to implement any changes, but if someone else wants to they can feel free ofc.